packages/vm/src/evm/evm.ts

```diff
@@ -572,7 +572,7 @@ export default class EVM {
    * Returns code for precompile at the given address, or undefined
    * if no such precompile exists.
    */
-  getPrecompile(address: Address): PrecompileFunc {
+  getPrecompile(address: Address): PrecompileFunc | undefined {
```

Comment:
I have changed this function sig, because it is incorrect anyways. Our internal logic calls this method, if `undefined` is returned we run a normal call, otherwise we run the precompile. (Also per the comment above)